### PR TITLE
CHANGELOG: release 0.106.2

### PR DESCRIPTION
**Pull request by AnnaShaleva**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,39 @@
 
 This document outlines major changes between releases.
 
+## 0.106.2 "K" (13 Jun 2024)
```
Comment (AnnaShaleva) [inline CHANGELOG.md:5]: It's hard this time, I tried to think about something release-related.

Comment (roman-khimov) [reply CHANGELOG.md:5]: I've only got two other "proper" possibilities ("Karstification" already taken):
 * Kation
 * Keratinization

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,39 @@
...
+
+A 3.7.5-compatible version introducing new Domovoi hardfork that brings two bug-fixes
```
Comment (roman-khimov) [inline CHANGELOG.md:7]: two fixes to the protocol

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,39 @@
...
+ * distinguish log level for various node peer disconnection reasons (#3469)
+
+Improvements:
+ * ensure NeoFS nodes are configured when processing NeoFS oracle requests (#3455)
+ * NeoFS SDK dependency upgrade (#3483)
+ * ensure System.Runtime.GetNotifications handler can't break the MaxStackSize
+   constraint before and after Domovoi hardfork (#3485)
+
+Bugs fixed:
+ * remove deployed contract script from wallet's account (#3470)
```
Comment (roman-khimov) [inline CHANGELOG.md:34]: BTW, it's a bit more appropriate to describe bugs rather than fixes here.

File: CHANGELOG.md (hunk 1 above)
Comment (AnnaShaleva) [reply CHANGELOG.md:5]: > Keratinization

That was my option too, so decided.

File: CHANGELOG.md (hunk 3 above)
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed! It was like that before, but sometimes I forget about it.

File: CHANGELOG.md (hunk 2 above)
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**